packages/story-editor/src/app/story/useStoryReducer/actions.js

```diff
@@ -119,12 +119,12 @@ const updateElementById =
       payload: { elementIds: [elementId], properties },
     });
 
-const duplicateElementById =
+const duplicateElementsById =
   (dispatch) =>
-  ({ elementId }) =>
+  ({ elementIds }) =>
     dispatch({
       type: types.DUPLICATE_ELEMENT_BY_ID,
```

Comment:
Let's change this as well then everywhere for consistency:

```suggestion
      type: types.DUPLICATE_ELEMENTS_BY_ID,
```